ui/raidboss/data/03-hw/raid/a8s.js

```diff
@@ -301,22 +315,22 @@
       alertText: function(data, matches) {
         if (data.me == matches.target) {
           return {
-            en: 'Shared Tankbuster on YOU',
-            de: 'geteilter Tankbuster auf DIR',
+            en: 'Shared Tank buster on YOU',
```

Comment:
Why are you changing the English and German versions?

Reply:
I just looked with the respons.js file
![responses](https://user-images.githubusercontent.com/61565834/82714022-49298100-9c8d-11ea-872f-5f1c28e6829a.PNG)
I must restored ?

Reply:
I am sorry to balk here, but it would be helpful for me if you don't make incidental extra changes when you're making a larger change.  This pull request is "raidboss: alex midas fr" and so I expected to see french translations, but then there's a bunch of English and German changes, which are kind of confusing to see.

I think if you want to change the English "Shared Tankbuster" to "Shared Tank **B**uster" or to split the German, it'd be better to do that in a separate pull request, where you make that change across all of the files that have that phrase, rather than just changing this one here on its own, inconsistently.

Reply:
Yes you are right. I have no problem with that, your comments are very relevant. I want to do too much. I will restore that.